docs/rum_collection/crash_reporting.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Crash Reporting (beta)
+kind: documentation
+further_reading:
+  - link: "https://github.com/DataDog/dd-sdk-ios"
+    tag: "Github"
+    text: "dd-sdk-ios Source code"
+  - link: "/real_user_monitoring"
+    tag: "Documentation"
+    text: "Datadog Real User Monitoring"
+---
+
+Enable iOS crash reporting and error tracking to get comprehensive crash reports, and error trends in RUM UI. With this beta feature, you get access to 
+
+ - Aggregated iOS crash data and RUM crash attributes
+ - Desymbolicated iOS error reports
+ - Trend analysis with iOS error tracking
+
+This guide follows the setup for iOS crash reporting and error tracking in the following steps
+ - Add crash reporting
+ - Desymbolicate error reports
+ - Verify setup
+
+### Add crash reporting 
+
+If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [iOS RUM setup documentation][2]. Upgrade to [dd-sdk-ios v1.7.0+][3] to get access to iOS crash reporting and error tracking. Update your initialization snippet to include crash reporting:
+
+
+```
+Datadog.initialize(
+    appContext: .init(),
+    trackingConsent: .pending,
+    configuration: Datadog.Configuration
+    .builderUsing(
+        rumApplicationID: "APP_ID",
+        clientToken: "CLIENT_TOKEN",
+        environment: "ENVIRONMENT"
+    )
+    .trackUIKitActions()
+    .trackUIKitRUMViews()
+    .enableCrashReporting(using: DDCrashReportingPlugin())
+    .build()
+)
+Global.rum = RUMMonitor.initialize()
+```
+
+### Desymbolicate reports
+
+If your mobile iOS source code is symbolicated, upload your dSYM file to Datadog so that your different stack traces can be desymbolicated. For a given error, you can get access to the file path, the line number, as well as a code snippet for each frame of the related stack trace.
+
+#### Install Datadog CI
+
+If you haven't already, install [Datadog CI][4] through NPM or Yarn. The package is under [@datadog/datadog-ci][5]. 
+
+```sh
+# NPM
+npm install --save-dev @datadog/datadog-ci
+
+# Yarn
+yarn add --dev @datadog/datadog-ci
+```
+
+If you need `datadog-ci` as a CLI tool instead of a package, you can run it with [`npx`][6] or install globally:
+
+```sh
+# npx
+npx @datadog/datadog-ci [command]
+
+# NPM install globally
+npm install -g @datadog/datadog-ci
+
+# Yarn v1 add globally
+yarn global add @datadog/datadog-ci
+```
```

Comment:
i think we can keep it shorter than that.
most of the cases `npx` should be enough (`npm install/yarn add` can even cause problems/confusion)

instead of `Install Datadog CI` + `Upload dSYM files` + `Verify setup`, we can put a `Using Datadog CI` section:
```
# Using Datadog CI

export DATADOG_API_KEY="<API KEY>"

// if you have a zip file containing dSYMs
npx @datadog/datadog-ci dsyms upload appDsyms.zip

// if you have a folder containing dSYMs
npx @datadog/datadog-ci dsyms upload /path/to/appDsyms/

For more info, please visit https://github.com/DataDog/datadog-ci/blob/master/src/commands/dsyms/README.md
```

what do you think? @maxep @ncreated

Reply:
Does 
```
npx @datadog/datadog-ci dsyms upload ...
``` 
come with any overhead when compared to following?
```
datadog-ci dsyms upload ...
```
I'm wondering if using `npx` from Build Phase would be same performant as using the tool directly (after installing it globally with `npm`).

Reply:
even if there is overhead, it is negligible (i don't notice it).
but one of our iOS customers had problems with installing it globally due to a permission issue in their local. 
that's why i think `npx` way could be the best option for iOS devs.